src/manifest.json

```diff
@@ -9,28 +9,54 @@
     "gecko": {
       "id": "{957509b1-217a-46c7-b08b-f67d08d53883}",
       "strict_min_version": "115.0",
-      "strict_max_version": "130.*"
+      "strict_max_version": "115.*"
```

Comment:
why 115? any particular reason ? I've set for development TB 130

Reply:
We know the add-on works in TB115. The legacy -> WebExtension conversion should therefore be done in 115, to be sure we are not hit by bugs which are due to internal changes from 115 to 128.

I had a case where a partially updated add-on stopped working, and we thought we did something wrong in the update process legacy -> WebExtension, but it was not working because of an internal change 115 -> 128. That took us a while to find out.